### DISCUSSION TITLE
【公益】图书捐赠 - ApacheCN

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#222] [Ideas]

**Comment by moc1546218029** [comment]
领取: 《算法导论》 《重构大数据统计》 《数据挖掘概念与技术》
姓名: 王恒
QQ: 1546218029
电话: 15843303136
地址: 吉林省长春市临河街东安加州枫景小区22栋二单元
备注: 邮费会付给您，拜托了，谢谢大家！

---
金额: 27元
快递名: 百世快递
订单号: 70750903644069
更新时间: 2018-12-29
寄送人：ApacheCN-小瑶

**Comment by AllenZYJ** [comment]
领取: 《统计学习方法》 《大数据挖掘》
姓名: AllenZYJ
QQ: 2533524298
电话: 15732568732
地址: 福建省漳州市平和东大街124号
备注: 感谢！邮费会付给您

---
金额: 15元
快递名: 百世快递
订单号: 70750909644311
更新时间: 2018-12-29
寄送人：ApacheCN-小瑶

**Comment by ABC1101904021** [comment]
领取: 《面试和算法心得》 《大数据分析》
姓名: ABC1101904021 
QQ: 1101904021
电话: 17633909850
地址: 贵州省贵阳市花溪区贵州大学北校区
备注: 感谢博主的活动

---
金额: 15元
快递名: 百世快递
订单号: 70750905644073
更新时间: 2018-12-29
寄送人：ApacheCN-小瑶

**Comment by wangzhenlonga** [comment]
领取: 《大数据商业分析》
姓名: 王振龙
QQ: 1262120514
电话: 17520464522
地址: 广东省深圳市龙华区龙华街道狮头岭一区1栋四楼411号
备注: 谢谢您，您的这种优良作风值得所有人学习！感谢无私奉献！邮费当然自己出哈哈😄

---
金额: 15元
快递名: 百世快递
订单号: 70750900644315
更新时间: 2018-12-29
寄送人：ApacheCN-小瑶

**Comment by chauncygu** [comment]
谢谢您，觉得您很赞！
想要的书已经被领走了，您的精神值得我们学习！

**Comment by jiangzhonglian** [comment]
> @moc1546218029
> 请问 怎样付邮费？ 

首页的绿帽子二维码，你直接根据你的邮费，发过去就行
